src/options/config.rs

```diff
@@ -0,0 +1,32 @@
+use crate::theme::UiStyles;
+use serde_yaml;
+use std::path::PathBuf;
+
+#[derive(Debug, Default, Eq, PartialEq)]
+pub struct ThemeConfig {
+    // This is rather bare for now, will be expanded with config file
+    pub location: ConfigLoc,
```

Comment:
is this pub really necessary ?

Reply:
hard to say so early but probably not. i'll change it :+1: